lib/plugins/browser-sync.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const loaderFeatures = require('../features');
+
+/**
+ * @param {Array} plugins
+ * @param {WebpackConfig} webpackConfig
+ * @returns {void}
+ */
+module.exports = function(plugins, webpackConfig) {
+
+    if (!webpackConfig.useBrowserSync) {
+        return;
+    }
+
+    loaderFeatures.ensurePackagesExist('browsersync');
+    const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+
+    let browserSyncOptions = {};
```

Comment:
`const` instead of `let`.

I would also have added the default settings there instead of an empty object: in other callbacks the objects that the user modifies are already set (which allows them for example to modify an option based on another one), which won't be the case here.

Reply:
you are right. I will move initialization there. 

Regarding `let` or `const` here, the object is changed in callbacks, would not this bring troubles ?

Reply:
Using `const` won't be an issue if you don't want to re-assign the variable later (modifying its properties is OK though):

```js
const obj1 = {};

const callback = (param) => {
  param.test = false;
};

// allowed, then obj1.test === true
obj1.test = true;

// allowed, then obj1.test === false
callback(obj1); 

// not allowed (Assignment to constant variable)
obj1 = { test: false }
```